Description: Skip Normaliz package for now.
 It's broken in unstable due to e-antic version bump:
 https://ci.debian.net/data/autopkgtest/testing/amd64/n/normaliz/5446108/log.gz
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug-Debian: https://bugs.debian.org/960614
Last-Update: 2020-05-14

--- a/M2/Macaulay2/packages/=distributed-packages
+++ b/M2/Macaulay2/packages/=distributed-packages
@@ -46,7 +46,6 @@
 Polymake
 gfanInterface
 PieriMaps
-Normaliz
 Posets
 XML
 OpenMath
@@ -74,7 +73,6 @@
 Parametrization
 Serialization
 NAGtypes
-NormalToricVarieties
 DGAlgebras
 Graphs
 GraphicalModels
@@ -99,7 +97,6 @@
 Triplets
 CompleteIntersectionResolutions
 MCMApproximations
-MultiplierIdeals
 InvariantRing
 QuillenSuslin
 EnumerationCurves
